src/OpenLoco/src/GameCommands/Track/CreateTrainStation.cpp

```diff
@@ -97,21 +87,6 @@ namespace OpenLoco::GameCommands
             }
             return std::make_pair(NearbyStationValidation::requiresNewStation, StationId::null);
         }
-        else
```

Comment:
Remove altogether rather than behind a cheat?

Reply:
Personally, I'd be happy to get rid of the limit entirely. I think 8 tiles is just rather short. Alternatively, perhaps we could double the limit to -say- 16 tiles, along with a cheat to remove the limit entirely?